src/Microsoft.AspNet.Mvc.Extensions/Formatters/JsonOutputFormatter.cs

```diff
@@ -11,7 +10,18 @@
 
 namespace Microsoft.AspNet.Mvc
 {
-    public class JsonOutputFormatter : OutputFormatter, IJsonOutputFormatter
+    /// <summary>
+    /// An output formatter that specializes in writing JSON content.
+    /// </summary>
+    /// <remarks>
+    /// The <see cref="JsonResult"/> class filter the collection of
+    /// <see cref="MvcOptions.OutputFormatters"/> and use only those of type
+    /// <see cref="JsonOutputFormatter"/>.
+    ///
+    /// To create a custom formatter that can be used by <see cref="JsonResult"/>, derive from
+    /// <see cref="JsonOutputFormatter"/>.
```

Comment:
Yeah, but you might as well not add the `<remarks />` since you're about to make it invalid.